core-primitives/hashing/Cargo.toml

```diff
@@ -4,7 +4,10 @@ version = "0.9.0"
 authors = ["Integritee AG <hello@integritee.network>"]
 edition = "2021"
 
-
 [dependencies]
 # substrate
 sp-core = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.28" }
+
+[features]
+default = []
+std = []
```

Comment:
Default does usually imply std. I would not go against this practice.

Reply:
That was on purpose, because the dependencies do not use this crate with `default-features=false` - but maybe it's worth changing that in all dependencies for consistency’s sake 👍

Reply:
Yes, this is also the (almost) official practice outside our repositories and substrate.